client/coniksclient/internal/cmd/lookup.go

```diff
@@ -61,6 +67,10 @@ var lookupCmd = &cobra.Command{
 					fmt.Println("Cannot get the key from the response, error: " + err.Error())
 				} else {
 					fmt.Println("Success! Key bound to name is: [" + string(key) + "]")
+					bindings[name] = string(key)
+					if err := writeKeys(bindings, conf.KeyStoragePath); err != nil {
+						fmt.Println("Cannot save the key to the file. Error: " + err.Error())
+					}
```

Comment:
Should we also have a `CheckBindingsDiffer` case? It seems like this would be a result that the client might get if the server has inserted a fake key.